### PR TITLE
Add Film model

### PR DESCRIPTION
**Pull request by mateuszCabala95**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```
Comment (jkopyto) [inline .idea/.gitignore:1]: dodaj .idea do .gitignore i usuń folder z pr

File: src/Models/AgeCategory.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+enum AgeCategory {
+    "NoRestriction",
+    "NoUnder13",
+    "NoUnder16",
```
Comment (jkopyto) [inline src/Models/AgeCategory.ts:4]:
```suggestion
    "Over16",
```

tak samo `"Over13"`

File: src/Models/FilmTypes.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+enum FilmTypes {
```
Comment (jkopyto) [inline src/Models/FilmTypes.ts:1]:
```suggestion
type FilmTypes = "comedy" | "horror" | "action" | "anime"
```

File: src/Models/Film.interface.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import FilmTypes from "./FilmTypes"
```
Comment (jkopyto) [inline src/Models/Film.interface.ts:1]: nazwa folderu models z małej litery

**Review by jkopyto — CHANGES_REQUESTED** [review]

File: src/Models/Film.interface.ts [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+export default interface IFilm {
+    title: string
+    producent: string
+    mainActors: string[]
+    type: FilmTypes
+    trailer?: string
+    filmLength: number
+    ageCategory: AgeCategory
+    is3D: boolean
+}
```
Comment (R4fau) [inline src/Models/Film.interface.ts:13]: może by dodać language / dubbing / subtitles ?

File: .idea/.gitignore (hunk 1 above)
Comment (mateuszCabala95) [reply .idea/.gitignore:1]: done

File: src/Models/AgeCategory.ts (hunk 2 above)
Comment (mateuszCabala95) [reply src/Models/AgeCategory.ts:4]: Done, change  from enum to type

File: src/Models/FilmTypes.ts (hunk 3 above)
Comment (mateuszCabala95) [reply src/Models/FilmTypes.ts:1]: Done

**Pull request merged**